docs/virtual_machines/network_binding_plugins.md

```diff
@@ -42,9 +42,9 @@ and integrate it into Kubevirt in a modular manner.
 Kubevirt is providing several network binding plugins as references.
 The following plugins are available:
 
-- slirp [v1.1.0]
-- passt [v1.1.0]
-- macvtap [v1.1.1]
+- [passt](net_binding_plugins/passt.md) [v1.1.0]
+- [macvtap](net_binding_plugins/macvtap.md) [v1.1.1]
+- [slirp](net_binding_plugins/slirp.md) [v1.1.0]
```

Comment:
Seems that "## Available network binding plugins" section is now redundant.

Reply:
Done